roles/openshift_health_checker/openshift_checks/package_version.py

```diff
@@ -9,6 +9,19 @@ class PackageVersion(NotContainerizedMixin, OpenShiftCheck):
     name = "package_version"
     tags = ["preflight"]
 
+    openshift_to_ovs_version = {
+        "3.6": "2.6",
+        "3.5": "2.6",
+        "3.4": "2.4",
```

Comment:
Are origin versions like 1.x being translated to 3.x somewhere? I don't see where that would happen.